app/javascript/dashboard/components/widgets/conversation/contextMenu/menuItem.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <div class="menu" role="button" @click.stop="$emit('click')">
+  <div
```

Comment:
The colors here felt a bit light after the Foundation CSS changes. Update it here. 

| Before | After |
| -- | -- |
|  <img width="207" alt="Screenshot 2024-03-03 at 7 53 20 PM" src="https://github.com/chatwoot/chatwoot/assets/2246121/a18a6d79-c9cc-4e86-86e8-f681f87fda6a"> | <img width="209" alt="Screenshot 2024-03-03 at 7 52 47 PM" src="https://github.com/chatwoot/chatwoot/assets/2246121/25205a08-1671-4256-853e-4e47d202ef89"> |